src/plugins/syntax-highlighting/index.ts

```diff
@@ -0,0 +1,96 @@
+import { PluginController } from "../PluginController";
+import { generateBracketPairColorizationCSS } from "./bracket-pair-colorization";
+import { Config, configList } from "./config";
+import "./index.less";
+
+// assumes valid input;
+export function hex2rgb(hex: string): [number, number, number] {
+  return [
+    parseInt(hex.slice(1, 3), 16),
+    parseInt(hex.slice(3, 5), 16),
+    parseInt(hex.slice(5, 7), 16),
+  ];
+}
```

Comment:
You may be interested in `parseCSSHex` from GLesmos `colorParsing` (which could probably be moved to a utils somewhere). Such a refactor is probably only helpful if invalid input is possible.

Reply:
Invalid input should be impossible here so I think I'm fine on that front.